src/host/main.cpp

```diff
@@ -107,6 +107,13 @@ int main(int argc, char** argv)
     "Path to which the node's node-to-node address (including potentially "
     "auto-assigned port) will be written. If empty (default), write nothing");
 
+  std::string node_client_host = {};
+  app.add_option(
+    "--node-client-host",
+    node_client_host,
+    "Host on which to connect to for commands sent to other nodes. If "
```

Comment:
```suggestion
    "Host on which to bind to for commands sent to other nodes. If "
```

I think this would be better as "address", it would be more consistent with our other flags, "host" is a bit overloaded.

Reply:
It seems that we already define an address as a `host:port` pair. In this specific case, we don't want operators to specify a port as _many_ outgoing n2n connections are created on the node. 

(Note: I actually tried to use a hardcoded port here to see how it would fail but to my surprise, it didn't. I believe this is because libuv uses [`SO_REUSEADDR`](https://github.com/libuv/libuv/blob/v1.x/src/unix/tcp.c#L164)).

Reply:
Good point, how about "interface" then?